### PR TITLE
Add splash screen feature

### PR DESCRIPTION
**Pull request by Cosmosha**
The following requirement and modification was implemented;

- **Make splash screen component basic structure.**

- **Update App.js route methods**

- **Update Splash.js page**

### CODE REVIEW

**Review by MoyasiGinko — CHANGES_REQUESTED** [review]
### Hey @Cosmosha 👋 
Good job implementing the splash screen, but there are some issues that needs to be worked on before you can move forward.



### Required Changes: ♻️ 
- Please follow the working directory convention, there can't any `CSS` folder inside the `components`. if there's any custom styling you wish to add, please add them inside the `styles` directory. We want to keep our working directory organized and professional.
- Please check your `App.js` and `Splash.jsx` file. you are importing the `SplashScreen.js` inside the `Splash.jsx` but in the `App.js` you are also importing the `SplashScreen.js`. Please fix this, importing the `Splash.jsx` into the `App.js` instead of `SplashScreen.js`.

**Comment by Cosmosha** [comment]
Hi @MoyasiGinko,
Thank you for making time to review my PR, regarding the 1st point you mentioned will check and do as such, however concerning the App.js  please check it well, all importation is coming from the pages folder, which includes the Splash.jsx. I will be grateful if you will show the portions of the code for modification next time. All the same, thank you for the feedback.

**Comment by ibugithub** [comment]
Hey Brother @Cosmosha I see, you did a great job by changing your code suggested by Moyasi. Thanks so much for that. 
I've noticed one thing you put the styles for the splash in the index.css file which is good. 
What if we could create a new styles folder and then create a file called splash.css inside it? 
Then we can find the styles related to splash so easily. right? Let me know what you think.

File: src/index.css [hunk 1]
```diff
@@ -21,6 +21,12 @@ body {
   height: 100vh;
 }
 
+.splash-container > .splash {
+  border: 1px solid #000;
+  border-radius: 2rem;
+  box-shadow: 8px 14px 8px grey;
+}
```
Comment (ibugithub) [inline src/index.css:28]: Hey Brother @Cosmosha I see, you did a great job by changing your code suggested by Moyasi. Thanks so much for that.
I've noticed one thing you put the styles for the splash in the index.css file which is good.


What if we could create a new styles folder and then create a file called splash.css inside it?
Then we can find the styles related to splash so easily. right? Let me know what you think.

**Review by ibugithub — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @Cosmosha , 

Good job so far!
There are some issues that you still need to work on.






## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Comment by Cosmosha** [comment]
Hi @ibugithub 
Bro,  thanks for making time to review my PR, however, please can you be specific about the areas you were suggesting for the change?

**Comment by ibugithub** [comment]
Hi @Cosmosha I'm sorry but Brother check the video on the layout page when I click on every nav item for example Reserve, My Reservation you'll see not found === routing in app.js is not correctly configured. I hope now you've got the point.

**Review by MoyasiGinko — APPROVED** [review]
### Hey @Cosmosha 👋 
Good job so far applying the required changes. There's nothing else to say other than its time to merge it.

### Status: APPROVED ✅

**Pull request merged**